app/scripts/lib/errors.js

```diff
@@ -136,6 +136,16 @@ define(function (require, exports, module) {
       return error.errno === code;
     },
 
+    /**
+     * Check if an error was created by this module.
+     *
+     * @param {object} error - error to check
+     * @returns {boolean} - true if from this module, false otw.
+     */
+    createdError: function (error) {
```

Comment:
> There's an is directly above this function.

Whoops. :blush:

> I'm not happy with the current name either, any other suggestions?

So one suggestion, maybe it's dirty, would be to overload the behaviour of `is` based on the presence of that second argument.

Failing that, I'm not sure I have any better ideas tbh. `isInstance`?